### PR TITLE
[WIP] Fetch Subscription Watch info dynamically when running superkey workflow

### PR DESCRIPTION
**Pull request by lindgrenj6**
As discussed in #353, we should be fetching account number + policy JSON at runtime rather than going off of static values, this PR is that work.

\# TODO: 
- [x] Service class to fetch data 
- [x] Key off of application type and whether we should fetch it dynamically or go off of `AppMetaData`

cc @infinitewarp

### CODE REVIEW

File: lib/sources/subscription_watch_info.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+module Sources
+  class SubscriptionWatchInfo
+    XRHID = Base64.strict_encode64({:identity => {:account_number => "12345", :user => {:is_org_admin => true}}}.to_json).freeze
```
Comment (lpichler) [inline lib/sources/subscription_watch_info.rb:3]: `account_number` should be hardcoded ?

Comment (lindgrenj6) [reply lib/sources/subscription_watch_info.rb]: Yeah this is just a dummy header to fetch from their Internal API, the acct # doesn't matter.  

Might move it out to a svc class haven't decided yet.

Comment (lpichler) [reply lib/sources/subscription_watch_info.rb]: 👍 thanks

**Comment by lindgrenj6** [comment]
Closing this due to doing it on the superkey side now here: https://github.com/RedHatInsights/sources-superkey-worker/pull/42

**Pull request closed without merging**